avaframe/com1DFA/com1DFACfg.ini

```diff
@@ -69,6 +69,11 @@ relThPercentVariation =
 # if a variation on relTh shall be performed add here +- absolute value and number of steps separated by $
 # for example relThRangeVariation=0.5$10
 relThRangeVariation =
+# if variation on relTh shall be performed using a normal distribution in number of steps,
+# value of buildType (ci95 value), min and max of dist in percent, buildType (ci95 only allowed),
+# support (e.g. 10000) all separated by $: e.g. normaldistribution$numberOfSteps$0.3$95$ci95$10000
+# if relThFromShp=True ci95 is read from shp file too
+relThDistVariation =
```

Comment:
why put normaldistribution at the beginning of normaldistribution$numberOfSteps$0.3$95$ci95$10000.
It is the only option no?

Reply:
so far yes, but already wanted to keep it flexibel if more distributions will be added